middlewares/stripPrefixRegex_test.go

```diff
@@ -73,18 +73,16 @@ func TestStripPrefixRegex(t *testing.T) {
 				actualPath = r.URL.Path
 				actualHeader = r.Header.Get(ForwardedPrefixHeader)
 			})
-
 			handler := NewStripPrefixRegex(handlerPath, testPrefixRegex)
-			server := httptest.NewServer(handler)
-			defer server.Close()
 
-			resp, err := http.Get(server.URL + test.path)
-			require.NoError(t, err, "%s: unexpected error.", test.path)
+			req := testhelpers.MustNewRequest(http.MethodGet, "http://localhost"+test.path, nil)
+			resp := &httptest.ResponseRecorder{Code: http.StatusOK}
+
+			handler.ServeHTTP(resp, req)
 
-			assert.Equal(t, test.expectedStatusCode, resp.StatusCode, "%s: unexpected status code.", test.path)
+			assert.Equal(t, test.expectedStatusCode, resp.Code, "%s: unexpected status code.", test.path)
```

Comment:
Yes, you can see sub-test names but not the log related to only one sub-test.
You can show all log (without the tests "headers") or nothing 😞 

Run tests manually it's not acceptable for me, I need a tool which run continuously my tests after each code changes.

I agree to "not pollute" our tests due to one IDE but it will be hard to me to analyze failed tests. 😞 😞 😞